src/eth.rs

```diff
@@ -180,6 +184,20 @@ impl std::cmp::PartialEq<str> for NodeOrRpcUrl {
     }
 }
 
+/// KiMap address. (simulation mode flag in process_lib?)
+const KIMAP: &str = "0x0165878A594ca255338adfa4d48449f69242Eb8F";
```

Comment:
IMO this should be a field in a Builder that we expose similar to the eth provider one. User brings their own CA.